Kickstarter-iOS/Views/Controllers/SettingsPrivacyViewController.swift

```diff
@@ -0,0 +1,131 @@
+import KsApi
+import Library
+import Prelude
+import Prelude_UIKit
+import SafariServices
+import Result
+import UIKit
+
+internal final class SettingsPrivacyViewController: UITableViewController {
+  internal let viewModel: SettingsPrivacyViewModelType = SettingsPrivacyViewModel()
+  fileprivate let dataSource = SettingsPrivacyDataSource()
+
+  internal static func instantiate() -> SettingsPrivacyViewController {
+   return Storyboard.SettingsPrivacy.instantiate(SettingsPrivacyViewController.self)
+  }
+
+  internal override func viewDidLoad() {
+    super.viewDidLoad()
+
+    self.tableView.dataSource = self.dataSource
+
+    self.tableView.tableHeaderView = UIView(frame: CGRect(x: 0, y: 0, width: 0, height: Styles.grid(4)))
+    self.tableView.tableHeaderView?.backgroundColor = .ksr_grey_100
+
+    self.viewModel.inputs.viewDidLoad()
+  }
+
+  internal override func bindStyles() {
+    super.bindStyles()
+
+    _ = self
+      |> UITableViewController.lens.view.backgroundColor .~ .ksr_grey_100
+      |> UITableViewController.lens.title %~ { _ in Strings.Privacy() }
+  }
+
+  internal override func bindViewModel() {
+    super.bindViewModel()
+
+    self.viewModel.outputs.reloadData
+      .observeForUI()
+      .observeValues { [weak self] user in
+        self?.dataSource.load(user: user)
```

Comment:
Hey @cdolm92, I think the reason the snapshot tests are failing (the label not being set) is because the table view isn't being reloaded after the dataSource is loaded. So you need a `self?.tableView.reloadData()` here and on [line 50](https://github.com/kickstarter/ios-oss/pull/403/files#diff-68ef9f446f155ed1595275b74d7fdac7R50).